<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editDialog</class>
 <widget class="QDialog" name="editDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>239</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>239</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>再次输入新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOri">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color: rgb(36, 118, 217);</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNow1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color: rgb(36, 118, 217);</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNow2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color: rgb(36, 118, 217);</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>12</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/Icon/Icon/config-white-48.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>-9</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEditOri</zorder>
  <zorder>lineEditNow1</zorder>
  <zorder>lineEditNow2</zorder>
  <zorder>buttonOk</zorder>
  <zorder>buttonCancel</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
